src/System.Net.Http/src/System/Net/Http/Managed/HttpConnectionKey.cs

```diff
@@ -13,16 +13,16 @@ internal struct HttpConnectionKey : IEquatable<HttpConnectionKey>
         public HttpConnectionKey(Uri uri)
         {
             UsingSSL = 
-                uri.Scheme == "http" ? false :
-                uri.Scheme == "https" ? true :
+                uri.Scheme == UriScheme.Http ? false :
+                uri.Scheme == UriScheme.Https ? true :
                 throw new ArgumentException("Invalid Uri scheme", nameof(uri));
```

Comment:
Yes, there are a ton of strings that need to be moved to the .resx, consolidated, clean up, revised, etc.  I've not done so here and don't plan to in this PR.